lib/circles/members/remove.js

```diff
@@ -0,0 +1,19 @@
+const _ = require("lodash");
+
+module.exports = function(req, res) {
+    // TODO assert that userId is member of circle
+    // TODO assert that authenticated user is member of or invited to circle
+    // TODO assert that userId is not the contactPerson of the circle
```

Comment:
If they *are* the contact person, the contact person field should just go blank :-)